code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -1514,6 +1514,25 @@
 
 	M.adjustToxLoss(1)
 
+/datum/reagent/vomit
```

Comment:
But then you can't have vomit smoke grenades and other memes

Also there are plenty of substances that are just mixtures of stuff that we already have, see: doctor's delight and other cocktails